config.py

```diff
@@ -155,6 +155,8 @@
 DATABASE_NAME = specific_parser['DATABASE'].get('database') \
                 or common_parser['DATABASE']['database']
 
+DATABASE_PORT = specific_parser['DATABASE'].get('port') or common_parser['DATABASE']['port']
```

Comment:
i found myself running multiple localhost postgres servers (different versions) on different ports, so just making the field customisable